app/controllers/api/base_controller.rb

```diff
@@ -1,8 +1,14 @@
 class Api::BaseController < ApplicationController
+  before_action :doorkeeper_authorize!, if: :doorkeeper_token
+  before_action :authenticate_with_oauth, if: :doorkeeper_token
   skip_before_action :require_ssl
 
   private
 
+  def authenticate_with_oauth
+    sign_in User.find_by_id(doorkeeper_token.resource_owner_id)
```

Comment:
Isn't this deprecated in favour of `User.find_by(id: doorkeeper_token.resource_owner_id)`?